### ISSUE TITLE
Error: undefined method 'psubscribe' for Redis::Client

### ISSUE THREAD

**Issue by robacarp** [jgaskins/redis#46]
In a project with redis installed like this: (from shard.lock)

```
version: 2.0
shards:
  redis:
    git: https://github.com/jgaskins/redis.git
    version: 0.7.0
```

I'm unable to use the psubscribe method:

```crystal
require "redis"

redis = Redis::Client.new

redis.psubscribe "*" do |on|
  on.message do |channel, message|
    puts message
  end
end
```

#subscribe works fine, however. 

```
In test_for_psubscribe.cr:5:7

 5 | redis.psubscribe "channel1", "channel2" do |subscription, connection|
           ^---------
Error: undefined method 'psubscribe' for Redis::Client
```

I poked briefly at the code in lib/redis and can see that it generates the function, but it's not getting imported up into Redis::Client like #subscribe is. I didn't look hard at finding a way to get the connection from the client to call psubscribe on, because it seemed like it got forgotten.

**Comment by jgaskins** [comment]
Ooh, thanks for the report. I think this was probably overlooked when I added `include Commands` to `Client` for uniformity in https://github.com/jgaskins/redis/commit/c77f7aa9c0fd5fdece3498a7bc448d3dc341aa99. Since it was no longer delegating to `Connection` via `method_missing`, it lost some methods that then needed to be added explicitly, and it looks like I missed `psubscribe`.

**Comment by jgaskins** [comment]
Fixed in a76ab13 and released in v0.7.1

**Comment by robacarp** [comment]
Thank you!
